src/middlewared/middlewared/api/v25_10_0/virt_device.py

```diff
@@ -101,8 +101,13 @@ class GPU(Device):
     vendorid: NonEmptyString | None = None
 
 
+class PCI(Device):
```

Comment:
Need to add these changes to the 25_04_0 directory since we want to backport this to RC.1